.env.example

```diff
@@ -0,0 +1,13 @@
+CDIP_API_ENDPOINT=https://sensors.api.stage.gundiservice.org/v2/observations/
+CDIP_ADMIN_ENDPOINT=https://api.stage.gundiservice.org
+GUNDI_API_BASE_URL=https://api.stage.gundiservice.org
+GCP_PROJECT_ID=cdip-stage-78ca
+KEYCLOAK_AUDIENCE=cdip-admin-portal
+KEYCLOAK_CLIENT_ID=cdip-integrations
+KEYCLOAK_CLIENT_SECRET=REPLACE_FOR_SECRET
+KEYCLOAK_ISSUER=https://cdip-auth.pamdas.org/auth/realms/cdip-dev
+LOOK_BACK_WINDOW_HOURS=720
```

Comment:
This one seems to be specific to a particular integration, then it shouldn't be in the template repo